src/JakobSteinn/HTMLFormValidator/Validator.php

```diff
@@ -0,0 +1,132 @@
+<?php
+
+namespace JakobSteinn\HTMLFormValidator;
+
+use DiDom\Query;
+use DiDom\Document;
+use Zend\InputFilter\Input;
+use Zend\InputFilter\InputFilter;
+use Zend\Validator\ValidatorPluginManager;
+
+class Validator
+{
+    /**
+     * HTML document that should contain a form
+     *
+     * @var DiDom\Document
+     */
+    private $DOMDocument;
+
+    /**
+     * Validation rules for the given form
+     * 
+     * @var array
+     */
+    private $rules = [];
+
+    /**
+     * The selector used to find user form
+     * 
+     * @var string
+     */
+    private $formSelector = '//form[1]/input';
+
+    /**
+     * Instantiate a new Validator instance
+     * 
+     * @param string $html 		The HTML to search through, to find a form
+     */
+    public function __construct($html, $uniqueFormSelector = '')
```

Comment:
use `null` as a default value: empty string is still a string, and may be ambiguous